pkg/clusterversion/clusterversion.go

```diff
@@ -90,3 +99,268 @@ func (builder *Builder) validate() (bool, error) {
 
 	return true, nil
 }
+
+// WithDesiredUpdateImage adds the desired image to the clusterversion struct.
+func (builder *Builder) WithDesiredUpdateImage(desiredUpdateImage string, force bool) *Builder {
+	if valid, err := builder.validate(); !valid {
+		builder.errorMsg = err.Error()
```

Comment:
Can be removed once https://github.com/openshift-kni/eco-goinfra/pull/475/files#r1644903566 is added

Reply:
removed, thanks